website.js

```diff
@@ -18,6 +18,7 @@ var clientID        = options.identity.clientId
 var secret          = options.identity.clientSecret
 var redirect        = options.identity.redirectUrl
 var io              = require('socket.io')(server);
+var ip;
```

Comment:
Found the answer with a simple Google search; `data.ip = socket.request.connection.remoteAddress;`